### DISCUSSION TITLE
Capital lock key not working with Spiral Linux Gnome desktop

### DISCUSSION THREAD

**Discussion by quadransmuralis** [SpiralLinux/SpiralLinux-project#98] [Q&A] [answered]
Hello,
I installed Spiral Linux with the Gnome desktop,but I cannot get the caps lock key to work.
Would anyone have any suggestions please?
Thx

**Comment by geckolinux** [comment, marked as answer]
Hi there, yep, just go into the Gnome Tweak tool and remove the keyboard language switching shortcut key setting.
